tests/check_consistent.py

```diff
@@ -184,6 +177,10 @@ def check_metadata() -> None:
                 for part in dep_version.split("."):
                     assert part.isnumeric(), f"Bad version '{part}' in dependency {dep}"
 
+        assert set(data.get("tool", [])).issubset({"stubtest"}), f"Unrecognised tool for {distribution}"
+        for key in data.get("tool", {}).get("stubtest", {}):
+            assert key in stubtest_keys, f"Unrecognised stubtest key {key} for {distribution}"
```

Comment:
... we don't need to hardcode the tool here:

```suggestion
        assert set(data.get("tool", [])).issubset(tool_keys.keys()), f"Unrecognised tool for {distribution}"
        for tool, tk in tool_keys.items():
            for key in data.get("tool", {}).get(tool, {}):
                assert key in tk, f"Unrecognised {tool} key {key} for {distribution}"
```